articles/stream-analytics/stream-analytics-parallelization.md

```diff
@@ -54,7 +54,7 @@ An *embarrassingly parallel* job is the most scalable scenario we have in Azure
 
 1. If your query logic depends on the same key being processed by the same query instance, you must make sure that the events go to the same partition of your input. For Event Hubs or IoT Hub, this means that the event data must have the **PartitionKey** value set. Alternatively, you can use partitioned senders. For blob storage, this means that the events are sent to the same partition folder. If your query logic does not require the same key to be processed by the same query instance, you can ignore this requirement. An example of this logic would be a simple select-project-filter query.  
 
-2. Once the data is laid out on the input side, you must make sure that your query is partitioned. This requires you to use **PARTITION BY** in all the steps. Multiple steps are allowed, but they all must be partitioned by the same key. Under compatibility level 1.0 and 1.1, the partitioning key must be set to **PartitionId** in order for the job to be fully parallel. For jobs with compatility level 1.2 and higher, custom column can be specified as Partition Key in the input settings and the job will be paralellized automoatically even without PARTITION BY clause.
+2. Once the data is laid out on the input side, you must make sure that your query is partitioned. This requires you to use **PARTITION BY** in all the steps. Multiple steps are allowed, but they all must be partitioned by the same key. Under compatibility level 1.0 and 1.1, the partitioning key must be set to **PartitionId** in order for the job to be fully parallel. For jobs with compatility level 1.2 and higher, custom column can be specified as Partition Key in the input settings and the job will be paralellized automoatically even without PARTITION BY clause. For event hub output the property "Partition key column" must be set to use "PartitionId".
```

Comment:
(previous typo): can you edit this to:  parallelized automatically. (instead of paralellized automoatically)

on the text: do we need to specify the "partition key column" for output with compat level 1.2? If no I will move this sentence before we talk about 1.2

Reply:
This is still required for 1.2 (maybe we can optimize it out but this is the case for now).

Reply:
Typo updated.